### PR TITLE
Add accesibility to Tabs

### PR DESCRIPTION
**Pull request by jvelezpo**
The Tabs can now be navigated using the keyboard by hitting tab

Fixes: https://github.com/auth0/cosmos/issues/923

[x] tests passes

### CODE REVIEW

**Comment by jvelezpo** [comment]
i dont know why `continuous-integration/travis-ci/pr` is not passing, all the tests pass as expected 

image

**Comment by francocorreasosa** [comment]
Hey @jvelezpo! Thank you for your contribution! There's a small issue in the build but is something we need to fix. Once we resolve this issue will get back to you :)

File: core/components/molecules/tabs/tabs.js [hunk 1]
```diff
@@ -92,6 +98,8 @@ class Tabs extends React.Component {
               onClick={() => this.changeTab(index)}
               key={index}
               selected={selectedIndex === index}
+              tabIndex="0"
```
Comment (francocorreasosa) [inline core/components/molecules/tabs/tabs.js]: Hey 👋 What is this `tabIndex` for? All tabs would have it hard set to `"0"`.

Comment (jvelezpo) [reply core/components/molecules/tabs/tabs.js]: Hey!!
https://stackoverflow.com/questions/4115953/where-do-tabindex-0-html-elements-end-up-in-the-tabbing-order
and https://www.w3.org/TR/html401/interact/forms.html#tabbing-navigation

`Elements that have identical tabindex values should be navigated in the order they appear in the character stream.`

Comment (francocorreasosa) [reply core/components/molecules/tabs/tabs.js]: Excellent! Thank you for the clarification.

**Comment by siddharthkp** [comment]
@jvelezpo Thanks for taking the time!

The build is failing because of: https://github.com/auth0/cosmos/issues/989

File: core/components/molecules/tabs/tabs.js [hunk 2]
```diff
@@ -80,6 +80,12 @@ class Tabs extends React.Component {
     }
   }
 
+  handleKeyPress(e, index) {
+    if (e.key === 'Enter') {
```
Comment (francocorreasosa) [inline core/components/molecules/tabs/tabs.js]: Should this be with space instead of enter?

**Comment by francocorreasosa** [comment]
Hey! Can you rebase your PR from our current `master`? So we can get it run from the CI and merge it. Thanks

**Comment by jvelezpo** [comment]
> Hey! Can you rebase your PR from our current `master`? So we can get it run from the CI and merge it. Thanks

I will do it tonight 👍

**Comment by francocorreasosa** [comment]
@jvelezpo Just merged the fix to our master. One more rebase and we'd be ok 👌

**Comment by siddharthkp** [comment]
Merging this into a local branch `accessible-tabs` so that CI can run. More info here: https://github.com/auth0/cosmos/issues/989

Here's the staging url: https://auth0-cosmos-accessible-tabs.now.sh/docs

**Pull request merged**